src/voting/SimpleGaugeVoterSetup.sol

```diff
@@ -14,7 +14,7 @@ import {PermissionLib} from "@aragon/osx/core/permission/PermissionLib.sol";
 import {PluginSetup} from "@aragon/osx/framework/plugin/setup/PluginSetup.sol";
 
 // these should be interfaces
-import {SimpleGaugeVoter} from "@voting/SimpleGaugeVoter.sol";
+import {SimpleGaugeVoterV1_1_0 as SimpleGaugeVoter} from "@voting/SimpleGaugeVoter_v1_1_0.sol";
```

Comment:
Wouldn't this create a bit of a mess regarding which version is life?
As in, I know this new version is factually better, but at the same time we have both options deployed and this seems as mixing versions.

Reply:
It helps to avoid rewriting the code in all tests every time you update. You will still have the names in each contract. Potentially we can amend Deploy and Upgrade Factories to also store version IDs